extensions-api/src/main/scala/pl/touk/nussknacker/engine/migration/ProcessMigration.scala

```diff
@@ -10,7 +10,7 @@ trait ProcessMigration {
 
   def description: String
 
-  def migrateProcess(canonicalProcess: CanonicalProcess, category: String): CanonicalProcess
```

Comment:
Is this something that should mentioned in migration guide since this changes extensions-api?

Reply:
I checked and one of our clients uses this information so I reverted it and left a comment about that. I also extracted a ScenarioWithDetailsForMigrations class with minimal set of scenario fields used by migration mechanism. It is a one step towards making this API extracted and thanks to that we will know when we break the migration api